### DISCUSSION TITLE
Does Plausible support file download tracking via meta refresh?

### DISCUSSION THREAD

**Discussion by gravelld** [plausible/analytics#2705] [Anything Else]
I use a meta refresh tag to send files to people who want to download my software:

    <META http-equiv="refresh" content="3;URL=my-app.exe?uniqueId=4938492384">

Is this supported in file download tracking? A quick test suggested it isn't.

If you don't want to support this, can you suggest an alternative to track this?

**Comment by metmarkosaric** [comment]
thanks for the feedback @gravelld! i'll move this over to the "feature requests" category so we can better track the popularity of this use case which helps us prioritize our development efforts. 

we typically recommend our custom events for any type of use case that's not covered by our automated events extensions. see https://plausible.io/docs/custom-event-goals#want-to-trigger-custom-events-manually-with-custom-javascript

**Reply by gravelld** [reply]
Thanks! I assume from your action then it _isn't_ supported. 👍 I'll look into a custom event.

**Reply by metmarkosaric** [reply]
np! alternative approach (if you have the option to change it on your site) would be to change the anchor tag to `<a>` rather than to `<META>`. that may work with our current file downloads

**Reply by gravelld** [reply]
Oh yeah, I offer that too. But in most cases this is on a page _behind_ a "Download" button. i.e. "Download" links to a "Thanks for downloading" with "Your download should begin in a few seconds" page, which downloads via `<meta>` in the background, and has an additional "Can't wait?" `<a>` which does the download (and is correctly captured by Plausible currently).

This is a common pattern in downloadable software, see https://www.kalzumeus.com/2006/07/19/making-adwords-work-for-you/
